app/javascript/styles/components.scss

```diff
@@ -1203,7 +1200,7 @@
   }
 
   &.dropdown__right {
-    right: 0;
+    left: 160px;
```

Comment:
`.dropdown__right` should be positioned from right side, because contents may be wider. For example, Occitan language:

![image](https://user-images.githubusercontent.com/705555/27225899-41fe7ab4-52d7-11e7-8e64-b87b18144751.png)